src/lineup/RankingViewComponent.tsx

```diff
@@ -156,7 +156,7 @@ export function RankingViewComponent({
   }, [status, selectionAdapter, selectionAdapterContext, inputSelection, selections, parameters, prevParameters, onContextChangedCallback]);
 
   return (
-    <div ref={viewRef} className={`tdp-view lineup lu-taggle lu ${status !== 'success' && 'tdp-busy'}`}>
+    <div ref={viewRef} className="h-100">
```

Comment:
@dvzacharycutler What about the `${status !== 'success' && 'tdp-busy'}` that is now missing? :thinking:

Reply:
Reprovisyn wasn't using it anyways, had their own tdp-busy setup based on the actual data loading, so maybe it shouldn't be here? But convinced myself just now its fine, pushed a change.